### PR TITLE
Improve upon gaps with `teams` commands

### PR DESCRIPTION
**Pull request by rocktavious**

### CODE REVIEW

File: src/cmd/team.go [hunk 1]
```diff
@@ -53,9 +64,9 @@ var createMemberCmd = &cobra.Command{
 
 var contactType string
 var createContactCmd = &cobra.Command{
-	Use:        "contact TEAM_ID|TEAM_ALIAS ADDRESS DISPLAYNAME",
+	Use:        "contact {TEAM_ID|TEAM_ALIAS} ADDRESS DISPLAYNAME",
 	Short:      "Add a contact to a team",
-	Long:       `Add a contact to a team`,
+	Example:    `opslevel create contact --type=slack my-team #general General`,
```
Comment (rocktavious) [inline src/cmd/team.go:69]: should we add more examples here?

Comment (eochoa260) [reply src/cmd/team.go:69]: Maybe one more wouldn't hurt.  Or having all the types in the help command but that might be harder to do.

**Review by eochoa260 — APPROVED** [review]
LGTM

File: src/cmd/team.go (hunk 1 above)
Comment (rocktavious) [reply src/cmd/team.go:69]: Hmm ya i think i can do this

**Pull request merged**